analyzers/src/SonarAnalyzer.VisualBasic/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace SonarAnalyzer.Extensions
 {
-    public static partial class SyntaxNodeExtensions
+    public static partial class SyntaxNodeExtensionsCSharp
```

Comment:
Shouldn't this be `SyntaxNodeExtensionsVbNet`?

Reply:
Ah, I only wanted to rename CSharp due to compilation errors and VS renamed too many files.. fixed to VB